public/project/css/nav.css

```diff
@@ -0,0 +1,45 @@
+/* Nav Container */
+.nav-container {
+  height: 70px;
+  width: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 2;
+}
+
+/* Nav Logo Top Left */
+.nav-container > img {
+  position: absolute;
+  top: 0;
+  left: 0;
+  padding: 30px;
+  width: 240px;
+}
+
+/* Hamburger */
+.ion-navicon {
+  color: white;
+  font-size: 40px;
+  right: 33px;
+  top: 24px;
+  transform: scale(1.3, 1);
+}
+
```

Comment:
When it comes to icons, you want to make it very clickable. When you add padding to an element, it increases its clickable range. So instead of doing top:24px, use top:0 right:0, and then apply padding: 24px 33px;. So now if some one goes to the very top right of the screen, they can still click